test/vaadin-combo-box-dropdown.html

```diff
@@ -18,6 +20,16 @@
     </template>
   </test-fixture>
 
+  <test-fixture id="in-modeless-dialog">
+    <template>
+      <vaadin-dialog modeless>
+        <template>
+          <vaadin-combo-box><template></template></vaadin-combo-box>
```

Comment:
Why the empty `<template>`? Use `<vaadin-combo-box items="[1,2,3]"></vaadin-combo-box>`

Reply:
removed empty `template`